es-models/gdc_from_graph/case.mapping.yaml

```diff
@@ -48,10 +48,18 @@ properties:
     type: keyword
   days_to_index:
     type: long
+  days_to_lost_to_followup:
+    type: long
   demographic:
     properties:
+      cause_of_death:
+        type: keyword
       created_datetime:
         type: keyword
+      days_to_birth:
+        type: long
+      days_to_death:
```

Comment:
There appear to be a lot of duplicates, I think entire property lists are duplicated between files and cases, I want to look into that further and see if we can make one source of truth for both (I think diagnoses is an example of that problem)